libkbfs/folder_block_manager.go

```diff
@@ -1022,8 +1026,17 @@ func (fbm *folderBlockManager) reclaimQuotaInBackground() {
 	}
 }
 
-func (fbm *folderBlockManager) getLastReclamationTime() time.Time {
-	fbm.lastReclamationTimeLock.Lock()
-	defer fbm.lastReclamationTimeLock.Unlock()
-	return fbm.lastReclamationTime
+func (fbm *folderBlockManager) getLastQRData() (time.Time, MetadataRevision) {
+	fbm.lastQRLock.Lock()
+	defer fbm.lastQRLock.Unlock()
+	return fbm.lastReclamationTime, fbm.lastQROldEnoughRev
```

Comment:
I guess this could conceivably return a new `lastReclamationTime` and an old `lastQROldEnoughRev`